src/ApparentHorizons/Tags.hpp

```diff
@@ -366,6 +366,24 @@ struct RicciScalarCompute : RicciScalar, db::ComputeTag {
   using return_type = Scalar<DataVector>;
 };
 
+/// The pointwise maximum of the Strahlkorper's intrinsic Ricci scalar
+/// curvature.
+struct MaxRicciScalar : db::SimpleTag {
+  using type = double;
+};
+
+/// Computes the pointwise maximum of the Strahlkorper's intrinsic Ricci
+/// scalar curvature.
+template <typename Frame>
+struct MaxRicciScalarCompute : MaxRicciScalar, db::ComputeTag {
+  using base = MaxRicciScalar;
+  using return_type = double;
+  static double function(const Scalar<DataVector>& ricci_scalar) {
+    return max(get(ricci_scalar));
+  }
```

Comment:
This is not absolutely trivial, so it should be tested.

Reply:
Maybe an easy way to test this would be to make a function in Test_Tags.cpp that makes a small DataVector and then tests that MaxRicciScalar::function() in fact returns the max?

Reply:
I think that would be fine.  You can put the DataVector in a DataBox and fully use the compute item if you want to be really thorough.